packages/component_library/lib/src/assets/imgs.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
```

Comment:
Does it make sense to name the folder assets in this case?

Reply:
To me it does, maybe we could move it to components. But I like it being in the assest folder

Reply:
This creates redundancy & makes code exploring/searching harder as there is already an assets folder in the root folder of each package. As you said moving it to components makes more sense.

Reply:
Done